tests/bgp/test_passive_peering.py

```diff
@@ -0,0 +1,198 @@
+'''
+
+This script is to test BGP passive peering on SONiC.
+
+'''
+
+import logging
+import pytest
+import time
+from tests.common.config_reload import config_reload
+from tests.common.helpers.constants import DEFAULT_NAMESPACE
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+bgp_config_sleeptime = 90
+peer_password = "sonic.123"
+wrong_password = "wrong-password"
+
+
+@pytest.fixture(scope='module')
+def setup(tbinfo, nbrhosts, duthosts, rand_one_dut_hostname, request):
+    # verify neighbors are type sonic
+    if request.config.getoption("neighbor_type") != "sonic":
+        pytest.skip("Neighbor type must be sonic")
+
+    duthost = duthosts[rand_one_dut_hostname]
+    dut_asn = tbinfo['topo']['properties']['configuration_properties']['common']['dut_asn']
+
+    lldp_table = duthost.shell("show lldp table")['stdout'].split("\n")[3].split()
+    tor1 = lldp_table[1]
```

Comment:
could you also change this name `tor1` here? it isn't necessarily tor

Reply:
@wenyiz2021 done.

Reply:
I don't see the change here?

Reply:
@wenyiz2021 My power got knocked out and interrupted the push it looks like.  It should be there now.